newIDE/app/src/MainFrame/index.js

```diff
@@ -2139,25 +2146,18 @@ const MainFrame = (props: Props) => {
         }
         previewState={previewState}
       />
-      <ClosableTabs hideLabels={!!props.integratedEditor}>
-        {getEditors(state.editorTabs).map((editorTab, id) => {
-          const isCurrentTab = getCurrentTabIndex(state.editorTabs) === id;
-          return (
-            <ClosableTab
-              label={editorTab.label}
-              key={editorTab.key}
-              id={`tab-${editorTab.key.replace(/\s/g, '-')}`}
-              active={isCurrentTab}
-              onClick={() => _onChangeEditorTab(id)}
-              onClose={() => _onCloseEditorTab(editorTab)}
-              onCloseOthers={() => _onCloseOtherEditorTabs(editorTab)}
-              onCloseAll={_onCloseAllEditorTabs}
-              onActivated={() => _onEditorTabActive(editorTab)}
-              closable={editorTab.closable}
-            />
-          );
-        })}
-      </ClosableTabs>
+      <DraggableClosableTabs
+        hideLabels={!!props.integratedEditor}
+        editorTabs={state.editorTabs}
+        onClickTab={(id: number) => _onChangeEditorTab(id)}
+        onCloseTab={(editorTab: EditorTab) => _onCloseEditorTab(editorTab)}
+        onCloseOtherTabs={(editorTab: EditorTab) =>
+          _onCloseOtherEditorTabs(editorTab)
+        }
+        onCloseAll={_onCloseAllEditorTabs}
+        onTabActive={(editorTab: EditorTab) => _onEditorTabActive(editorTab)}
```

Comment:
`onTabActivated`

(sadly, _onEditorTabActive was not a great name, because `on` should be followed from a verb: `_onEditorTabActivated`)